src/Blueprint/Blueprint/Models/PSParameterValueBase.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.Blueprint.Models
+{
+    public class PSParameterValueBase
```

Comment:
>PSParameterValueBase [](start = 17, length = 20)

Do we have display name also?

Reply:
Couldn't find display name in SDK models. Looked at ParameterValue and ParameterValueBase.

---
In reply to: [257122404](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257122404) [](ancestors = 257122404)

Reply:
Maybe they're missing from swagger, but yeah we can deal with that later.

---
In reply to: [258966822](https://github.com/Azure/azure-powershell/pull/8537#discussion_r258966822) [](ancestors = 258966822,257122404)